### ISSUE TITLE
Change AdminSet predicate to ons:primarySet

### ISSUE THREAD

**Issue by wickr** [OregonDigital/OD2#357]
### Descriptive summary

AdminSet predicate is set by default to `dct:isPartOf`, which we want to continue using for descriptive metadata.

https://github.com/OregonDigital/OD2/blob/master/config/initializers/hyrax.rb#L29
Change that to use `http://opaquenamespace.org/ns/primarySet`  for now. We may have a better solution later, but this will get rid of the immediate conflict with the properties and the form.

### Expected behavior

AdminSets function as expected and `dct:isPartOf` doesn't get populated with AdminSet resource URIs.

### Related work

https://github.com/samvera/hyrax/issues/1461

**Comment by CGillen** [comment]
First stab at this didn't look great. Just changing the given line seems to break something to do with assigning admin_set_id during work creation.
Simple debugging I tried were:
- Checking if form submits the same values. It does
- Checking if the environment that gets passed through the actor stack is the same. It is, `env['attributes']['admin_set_id']` is set, but `env['admin_set_id']` is not in both default and edited cases

Given those tests, there seems to be another assumption somewhere in Hyrax, or something we broke with a different edit we've done somewhere.
